github_pr.py

```diff
@@ -237,7 +237,7 @@ def main():
         github_update_pr(**args)
 
     gh = Github(args['token'])
-    if (('numberonly' in args) and not args['numberonly']) or (('noratelimit' in args) and not args['noratelimit']):
+    if not args['numberonly'] and not args['noratelimit']:
```

Comment:
I told you how to fix it before and you said you knew what you wanted. I thought you had tested it. We are now causing unnecessary churn in the master branch